packages/template-retail-react-app/app/hooks/use-auth-modal.js

```diff
@@ -59,8 +62,18 @@ export const AuthModal = ({
     }
 
     const handleLogin = async (data) => {
+        console.log('data', data)
         try {
-            await customer.login(data)
+            console.log('test')
+            loginRegisteredUser.mutate(
+                {username: data.email, password: data.password},
+                {
+                    onSuccess: (data) => {
+                        console.log('data', data)
```

Comment:
I guess this is still work-in-progress eh?

Reply:
oh sorry, this should not be here since it will be handled in a different ticket, I was trying to work on wishlist previously, hence I changed this to log in, but we have ticket to work on wishlist separately. I will revert this change. Thanks for catching that